### ISSUE TITLE
Error running on iOS 18, Xcode 16.0, : 'WidgetStateProperty' not found.

### ISSUE THREAD

**Issue by clintonksang** [srtraj/dropdown_textfield#59]
```
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/dropdown_textfield-1.1.0/lib/dropdown_textfield.dart:20:9: Error: Type
'WidgetStateProperty' not found.
```

Flutter Doctor -v
✓] Flutter (Channel stable, 3.19.0, on macOS 15.0 24A5264n darwin-arm64, locale en-KE)
    • Flutter version 3.19.0 on channel stable at /Users/myuser/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision bae5e49bc2 (4 months ago), 2024-02-13 17:46:18 -0800
    • Engine revision 04817c99c9
    • Dart version 3.3.0
    • DevTools version 2.31.1

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/myuser/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 16.0)
    • Xcode at /Applications/Xcode-beta.app/Contents/Developer
    • Build 16A5171c
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)

[✓] VS Code (version 1.90.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.90.0
    
    Steps to re-produce:
    
    Package fails on `flutter run`,

**Comment by shanuNanminda** [comment]
Getting same error on android.
```
../../../.pub-cache/hosted/pub.dev/dropdown_textfield-1.1.0/lib/dropdown_textfield.dart:20:9: Error: Type 'WidgetStateProperty' not found.
  final WidgetStateProperty<Color?>? fillColor;
        ^^^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/dropdown_textfield-1.1.0/lib/dropdown_textfield.dart:27:9: Error: Type 'WidgetStateProperty' not found.
  final WidgetStateProperty<Color?>? overlayColor;
        ^^^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/dropdown_textfield-1.1.0/lib/dropdown_textfield.dart:20:9: Error: 'WidgetStateProperty' isn't a type.
  final WidgetStateProperty<Color?>? fillColor;
        ^^^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/dropdown_textfield-1.1.0/lib/dropdown_textfield.dart:27:9: Error: 'WidgetStateProperty' isn't a type.
  final WidgetStateProperty<Color?>? overlayColor;
        ^^^^^^^^^^^^^^^^^^^
```

**Comment by rahulsharma9893** [comment]
same issue

**Comment by clintonksang** [comment]
![IMG_8559](https://github.com/srtraj/dropdown_textfield/assets/55794066/c26bfd83-fa47-4584-8e47-6b95ec1efb7e)

I updated the issue above, This should be a temporary fix as its being resolved by team

**Comment by srtraj** [comment]
Fixed in dropdown_textfield: ^1.2.0
